server/views/partials/viewCases.njk

```diff
@@ -183,6 +179,6 @@
         {% endif %}
         {% set licences = (licences.push(caseData), licences) %}
     {% endfor %}
-    {{ casesTable({licences: licences, tabType: params.tabType}) }}
+    {{ casesTable({licences: licences, tabType: params.tabType, probationView: false }) }}
```

Comment:
This macro is only used by the hardstop tabs - so always used for the prison view

Reply:
is `probationView` not another name for the `peopleOnProbation` tab rather than based on who is doing the viewing? Or am I misunderstanding your comment?

Reply:
Sorry I meant:
> This macro is only used by the tabs introduced by hardstop - so always used for the "in prison" section